src/components/LegendWrapper/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC, ReactNode } from 'react';
+import { useTheme, ThemeProvider } from 'styled-components';
+
+import defaultTheme from 'src/themes/cruk';
+import ErrorText from 'src/components/ErrorText';
+import Text from 'src/components/Text';
+
+import { StyledFieldset } from './styles';
+
+export type LegendWrapperProps = {
+  children?: ReactNode;
+  /** legend text */
+  legend: string;
+  /** error message text  */
+  errorMessage?: string;
+  /** flag for error styling */
+  hasError?: boolean;
+  /** when required (required) appears in label */
+  required?: boolean;
+  /** hint text */
+  hintText?: ReactNode | string;
+};
+
+export const LegendWrapper: FC<LegendWrapperProps> = ({

```

Comment:
`/** documentation about component  */ `using JS docs syntax appears on docs tab of story